target/snitch_cluster/cfg/snax.hjson

```diff
@@ -75,7 +75,7 @@
         },
     },
     // Templates.
-    snax_mac_core_template: {
+    snax_gemm_core_template: {
```

Comment:
This core is not enabled anywhere, and hence not properly tested?

Reply:
@JosseVanDelm we have a separate SW PR for this. Just to avoid cluttering PRs like you requested haha

Reply:
@rgantonio we don't need to wait for the software to try building this configuration, right? i.e. we can already build a snitch_cluster.vlt with the gemm accelerator, right?
I feel like right now the hardware that is introduced in this PR is not properly tested, I would still like to test it within this PR. It makes no sense to me to delay fixing issues arising from that when the software is added?